src/tests/end-to-end/common/element-identifiers/popup-page-element-identifiers.ts

```diff
@@ -5,7 +5,7 @@ export const popupPageElementIdentifiers = {
     startUsingProductButton: 'button.start-using-product-button',
     launchPad: '#new-launch-pad',
     launchPadItemTitle: '.launch-pad-item-title',
-    adhocPanel: '.ad-hoc-tools-panel',
+    adhocPanel: '#popup-container',
```

Comment:
This match the div id of `src\popup\popup.html`, i.e. it will match any content inside the popup (Launch Pad, Ad Hoc Panel...)
In other words, finding a div with this id doesn't mean that the ad hoc panel is up. Is that the intention?

Reply:
You are right. That was not the intention.